package.json

```diff
@@ -26,5 +26,10 @@
     "passport-local": "^1.0.0",
     "request": "^2.87.0",
     "underscore": "^1.9.1"
```

Comment:
Hey Patrick, what are these for?

Reply:
- passport-local is for authentication
- request is for creating http requests
- underscore.js according to the [docs](https://www.npmjs.com/package/underscore), is a utility-belt library for JavaScript that provides support for the usual functional suspects (each, map, reduce, filter...) without extending any core JavaScript objects.